### ISSUE TITLE
Lex v2 Imageresponsecard  not being rendered

### ISSUE THREAD

**Issue by Th3G4mbl3r** [aws-samples/aws-lex-web-ui#313]
I am using the Lex web ui to front a Lex v2 based bot. In my intent i have 2 slots. The 2nd slot is using an response card with buttons. However, the lex web UI does not render the response card with buttons on the browser, even though the developer tools clearly shows that the client has received a response from bot with contenttype imageresponse card.

Browser - Chrome 92.0

Response object - 

{"interpretations":[{"intent":{"confirmationState":"None","name":"signet-repair-jewellery-type","slots":{"jewellery-repair-document-type-slot":null,"jewellery-repair-slot":{"value":{"interpretedValue":"lab-created diamond","originalValue":"lab-created diamond","resolvedValues":["lab-created diamond"]}}},"state":"InProgress"},"nluConfidence":{"score":0.93}},{"intent":{"name":"Signet-Repair-Take-In-Intent","slots":{"take-in-repair-process-slot":null}},"nluConfidence":{"score":0.16}}],"messages":[{"contentType":"ImageResponseCard","imageResponseCard":{"buttons":[{"text":"Policy documents","value":"policy"},{"text":"Quick reference guides","value":"quick reference guide"},{"text":"Job aid documents","value":"job aid"}],"title":"What type of jewellery repair process document are you looking for?"}}],"sessionId":"ap-southeast-1:93b6812d-5ff4-4131-8593-fe82b2d542fe","sessionState":{"dialogAction":{"slotToElicit":"jewellery-repair-document-type-slot","type":"ElicitSlot"},"intent":{"confirmationState":"None","name":"signet-repair-jewellery-type","slots":{"jewellery-repair-document-type-slot":null,"jewellery-repair-slot":{"value":{"interpretedValue":"lab-created diamond","originalValue":"lab-created diamond","resolvedValues":["lab-created diamond"]}}},"state":"InProgress"},"originatingRequestId":"c3a019bf-56cb-4e32-9713-66d06d0b9a5e","sessionAttributes":{"accesstokenjwt":"some value","idtokenjwt":"esome value","refreshtoken":"somevalue","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36"}}}

**Comment by kalyanigaydhane** [comment]
kindly make change in messages attribute. it should be like below:
 "messages": [
    {
      "content": "Choose yes or no",
      "contentType": "PlainText"
    },
    {
      "contentType": "ImageResponseCard",
      "imageResponseCard": {
        "buttons": [
          {
            "text": "yes",
            "value": "yes"
          },
          {
            "text": "no",
            "value": "no"
          }
        ],
        "subtitle": "YESNO",
        "title": "YesNo"
      }
    }
  ]
  note : YESNO is a slot name.
  thanks

**Comment by LisaGotzian** [comment]
What worked for us is changing the `dialogAction` to "Close" instead of "ElicitIntent". The image response card still needs a plain text before it, but then it works.
```
'sessionState': {
                                            'dialogAction': {
                                                'type': 'Close',
                                            },
                                            'intent': {
                                                'name': intent_name,
                                                'state': 'Fulfilled',
                                            }
                                        },
                                        'messages': [
                                                        {
                                                        "contentType": "PlainText",
                                                        "content": "Super! Wie kann ich Ihnen sonst noch weiterhelfen?"
                                                        },
                                                        {
                                                            "contentType": "ImageResponseCard",
                                                            "imageResponseCard": {
                                                                "title": "Bitte wählen Sie eine Option:",
                                                                "buttons": [
                                                                    {
                                                                        "text": "noch ein Produkt suchen",
                                                                        "value": "product suchen"
                                                                    }, 
                                                                    {
                                                                        "text": "von vorn beginnen",
                                                                        "value": "von vorn beginnen"
                                                                    },
                                                                    {
                                                                        "text": "mit einer*m Mitarbeiter*in verbinden",
                                                                        "value": "mit einer*m Mitarbeiter*in verbinden"
                                                                    },
                                                                    {
                                                                        "text": "Chat schließen",
                                                                        "value": "Chat schliessen"
                                                                    }
                                                                    
                                                                ]
                                                            }
                                                        }   
                                                    ]
```

**Comment by bobpskier** [comment]
@LisaGotzian Just released version 0.19.3 of LexWebUi which should resolve displaying a response card when no message text is present. If still an issue please reopen. Thanks.
